examples/ord-min-max.js

```diff
@@ -0,0 +1,42 @@
+const {Max, Min, Ord} = require('../fantasy-monoids');
```

Comment:
Jop, Will add some unit tests and move to mocha. @SimonRichardson okay with you?